geoips/plugins/modules/readers/scat_noaa_winds_netcdf.py

```diff
@@ -232,3 +234,21 @@ def call(fnames, metadata_only=False, chans=None, area_def=None, self_register=F
     final_wind_xarrays["METADATA"] = wind_xarray[[]]
 
     return final_wind_xarrays
+
+
+def yeild_test_files():
+    """Yeild test xarray from test files for unit testing."""
+    filepath = (
+        environ["GEOIPS_TESTDATA_DIR"]
+        + "/test_data_scat/data/20230524_metopc_noaa*/*.nc"
+    )
+    filelist = glob(filepath)[:2]
+    tmp_xr = call(filelist)
+    if len(filelist) == 0:
+        raise NameError("No files found")
+    return tmp_xr
+
+
+def yeild_test_parameters():
+    """Yeild test data key for unit testing."""
+    return "WINDSPEED"
```

Comment:
I think yield_test_parameters could return something like:
{"dataset_name": "WINDSPEED",
"variable_name": "wind_speed_kts",
 "expected_min": pre_determined_min_val,
"expected_max": pre_determined_max_val,
"expected_mean": pre_determined_mean}

Reply:
Yeah, I think that sounds good. It could use the same values regardless of the input file or it could use values specific to the test data file. I think that, at least for now, this should just be the known min/max/mean of the known test data file.

Reply:
https://github.com/NRLMMD-GEOIPS/geoips/issues/390